<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>daily</class>
 <widget class="QWidget" name="daily">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Daily Quests</string>
  </property>
  <property name="windowIcon">
   <iconset theme="icon.ico"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Daily Quest</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="Day">
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Reset_Day">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Reset Quests</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day1">
      <property name="text">
       <string>XP Memoria Card 1 time</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day2">
      <property name="text">
       <string>XP Symphogear Card 1 time</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day3">
      <property name="text">
       <string>Clear 1 SP Quest</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day4">
      <property name="text">
       <string>Kill 10 Ennemies in Quest</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day5">
      <property name="text">
       <string>Kill 30 Ennemies in Quest</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day6">
      <property name="text">
       <string>Clear 1 Multi Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day7">
      <property name="text">
       <string>Join 1 Multi Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="Day8">
      <property name="text">
       <string>Join 3 Multi Battle</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Day5</sender>
   <signal>toggled(bool)</signal>
   <receiver>Day4</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Day8</sender>
   <signal>toggled(bool)</signal>
   <receiver>Day7</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
